compiler_opt/rl/trainer.py

```diff
@@ -77,6 +80,9 @@ def __init__(
         self._root_dir, flush_millis=summaries_flush_secs * 1000)
     self._summary_writer.set_as_default()
 
+    self._summary_log_interval = summary_log_interval
```

Comment:
nit: move it above between self._log_interval = log_interval and self._summary_interval = summary_interval